src/transformers/models/roberta/modeling_roberta.py

```diff
@@ -1067,6 +1067,8 @@ def set_output_embeddings(self, new_embeddings):
         output_type=MaskedLMOutput,
         config_class=_CONFIG_FOR_DOC,
         mask="<mask>",
+        expected_output="' Paris'",
```

Comment:
We should only have to add these values to the modeling file to make the doc test pass for other RoBERTa like tests :-)